.changeset/wicked-tomatoes-grow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@firebase/firestore': minor
```

Comment:
Can you also add `'firebase': minor`? Unfortunately changesets can't autodetect that a minor or major bump in a dependency ought to bump the main `firebase` package by the same increment so we have to add it manually.

Reply:
Done